app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,105 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductNotFountException;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+
+/**
+ * {@code ProductService} 클래스는 {@code Product}에 대한 비즈니스 로직을 처리합니다.
```

Comment:
자기 자신에 대한 링크를 걸 필요가 없습니다.
```suggestion
 * 상품에 대한 비즈니스 로직을 처리합니다.
```

Reply:
간결한 표현이 더 좋군요 .. !